README.md

```diff
@@ -58,7 +63,16 @@ actor ExampleStandard: Standard, HealthKitConstraint {
 
 
 Then, you can configure the [`HealthKit`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkit/healthkit-swift.class) module in the configuration section of your `SpeziAppDelegate`.
-You can, e.g., use [`CollectSample`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkit/collectsample) to collect a wide variety of HealthKit data types:
+
+There are several built-in configurations you can use:
+
+- [`CollectSample`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkit/collectsample) sets up data collection of HealthKit samples and delivers them to the functions defined in your app's `Standard` (as described above).
+- [`RequestReadAccess`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkit/requestreadaccess) defines which HealthKit sample types your app requires read access to, but does not set up data collection.
+- [`RequestWriteAccess`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkit/requestwriteaccess) defines which HealthKit sample types your app requires write access to.
+
+The example below sets up automated data collection for active energy burned, step count, push count, heart rate, and electrocardiograms. It also requests read access to the blood oxygen, which will prompt the user for authorization, but not handle automated collection of the data, which will need to be queried elsewhere in your code (see *Querying Health Data* below).
```

Comment:
I am wondering if we should flip the configuration elements and the standard constraint elements around; I think that would make more sense?

Reply:
I am thinking the same. There also needs to be a dead simple explanation of what a "Standard" actually is, although that should probably be in the Spezi or SpeziTemplateApplication repository, and linked here.

Reply:
I think the challenge is that you have to be a fairly advanced Swift developer to understand most of the jargon in these docs and you don't really need (or shouldn't need to be) that advanced to use the template/modules. That's not necessarily a bad thing, there just needs to be a beginner version for these core concepts.

Reply:
Agree; would be great to have some better examples and ways to explain it to someone who is new to the system! Thank you for working on this! 🚀

Reply:
@RealLast and I are now working on this in https://github.com/StanfordSpezi/learnspezi.

Reply:
That's great; thank you @vishnuravi and @RealLast 🚀